.github/workflows/static_check.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Use corepack
         run: corepack enable
       - name: Install node dependencies
-        run: yarn
+        run: yarn --immutable
```

Comment:
I think someone pushed some changes without correct `yarn.lock` file - so let's prevent it in future.